sbom-generator/action.yml

```diff
@@ -0,0 +1,26 @@
+name: 'generate-sbom'
+description: 'Generate SBOM for FreeRTOS libraries'
+inputs:
+  repo_path:
+    description: 'Path to repository folder containing manifest.yml to verify.'
+    required: false
+    default: ./
+  source_path:
+    description: 'Path to source code'
+    required: false
+    default: ./source
+  url:
```

Comment:
This should not be a parameter.
The github url can be obtained in the script using the GITHUB_REPOSITORY environment variable. Something along the lines of "https://github.com/$GITHUB_REPOSITORY"